### DISCUSSION TITLE
Latest release?

### DISCUSSION THREAD

**Discussion by Dergo32** [elishacloud/dxwrapper#223] [Q&A] [answered]
Hello,

I'm interested in using DxWrapper for Nascar Racing 2003 Season, in order to use DirectX 9 and to troubleshoot crashing issues I'm having. It looks like the source for DxWrapper has been updated a lot ever since the last release though. Can you release the latest version of DxWrapper?

Thank you

**Comment by elishacloud** [comment, marked as answer]
This is a duplicate of #152.  I am still regression testing it and fixing the bugs I find.  Here is a build of the current code:  [dxwrapper.zip](https://github.com/elishacloud/dxwrapper/files/12456141/dxwrapper.zip)
